lang/jvm/src/test/scala/com/wavesplatform/lang/ParserTest.scala

```diff
@@ -18,6 +18,11 @@ class ParserTest extends PropSpec with PropertyChecks with Matchers with ScriptG
     case Failure(_, _, _) => false
   }
 
+  def toString(expr: EXPR): String = expr match {
+    case CONST_LONG(x) => s"$x"
+    case _             => ???
+  }
```

Comment:
This method should add 0-N spaces (\n, \t, ' ') from left and right of the stringified expression.
Also, please delete an unnecessary test:
```scala
parse("10") shouldBe CONST_LONG(10)
```